samples/snippets/src/main/java/com/example/spanner/admin/archived/PgSpannerSample.java

```diff
@@ -203,7 +203,6 @@ static void createPostgreSqlDatabase(DatabaseAdminClient dbAdminClient, Database
       // Initiate the request which returns an OperationFuture.
       Database db = op.get();
       System.out.println("Created database [" + db.getId() + "]");
-      createTableUsingDdl(dbAdminClient, id);
```

Comment:
Why is this not needed anymore?

Reply:
https://github.com/googleapis/java-spanner/blob/main/samples/snippets/src/test/java/com/example/spanner/admin/archived/PgSpannerSampleIT.java#L138 This is already being called here. Its failing because duplicate table creation.

Reply:
Ack